configs/config.json.template

```diff
@@ -1,5 +1,8 @@
 {
+  "my_channel_id": "INSERT CHANNEL ID HERE",
+  "discord_channel_endpoint": "https://discordapp.com/api/v6/channels/",
   "discord_gateway_endpoint": "https://discordapp.com/api/v6/gateway",
+  "discord_user_endpoint": "https://discordapp.com/api/v6/users/@me/channels",
```

Comment:
This should probably be combined into a single `discord_api_endpoint` that all of these URLs share, and when we make the calls we can hard-code the rest of the URLs as strings. Right now we're repeating ourselves 3 times by having "https://discordapp.com/api/v6" repeated multiple times.

It's common to use URLs like `"{}/gateway".format(base_url)`, where all of the final path strings start with `/`.